.travis.yml

```diff
@@ -1,15 +1,16 @@
 language: php
 
 php:
-    - 5.6
-    - 7.0
     - 7.1
     - 7.2
-    - hhvm
+    - 7.3
 
 before_script: composer install --dev --prefer-source
 
 notifications:
   email: "douglas@usemarkup.com"
 
-dist: trusty
+dist: xenial
+
+script:
+  - vendor/bin/phpstan.phar analyse -c phpstan.neon -l 2 .
```

Comment:
I guess if you have the Makefile it might be worth calling it here also so `make analysis`

Reply:
I was thinking about it some time ago, but I've never seen this in CI configuration so I assumed this may not work everywhere.